x/twap/api_test.go

```diff
@@ -10,12 +10,27 @@ import (
 	"github.com/osmosis-labs/osmosis/v12/x/twap/types"
 )
 
+const (
```

Comment:
Can we write down comment on why these bool values are defined, and how they are defined?